packages/itwinui-css/src/utils/mixins.scss

```diff
@@ -142,3 +142,24 @@
     @content;
   }
 }
+
+// Hide borders and hover/focus effects, eg. input
+@mixin iui-invisible-borders {
+  border: none;
+  background: none;
+
+  &:focus,
+  &:focus-within,
```

Comment:
i think `focus-within` will do nothing here because there can be no child of input

Reply:
Removed.